db/migrate/20230613205213_create_comments.rb

```diff
@@ -0,0 +1,15 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :AuthorId
+      t.string :integer
+      t.string :PostId
+      t.string :integer
```

Comment:
- [ ] From the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) there is no column in the `Comment` table as `integer`. Kindly delete this column and ensure your migration follows the given ERD.